typescript/packages/plugins/minting-api/src/impl.ts

```diff
@@ -0,0 +1,116 @@
+import { z } from "zod";
+import type { Plugin, WalletClient } from "@goat-sdk/core";
+import { randomUUID } from "node:crypto";
+
+export function mintAPI(apiKey: string): Plugin<WalletClient> {
+    return {
+        name: "minting_api",
```

Comment:
I would move this whole plugin to `wallets/crossmint`. Checkout the faucet implementation: https://github.com/goat-sdk/goat/blob/main/typescript/packages/wallets/crossmint/src/faucet.ts

Then the way you get this plugin is by doing:

```typescript
const { mint } = crossmint(apiKey)
```

Reply:
Nice!